astropy/visualization/wcsaxes/coordinate_helpers.py

```diff
@@ -530,7 +530,11 @@ def get_axislabel(self):
         label : str
             The axis label
         """
-        return self.axislabels.get_text()
+        label = self.axislabels.get_text()
+        if self._auto_axislabel and not label:
```

Comment:
doesn't this imply that one cannot (intentionally !) set an empty label anymore ? This seems surprising enough that I'd call it a bug.

Reply:
Thanks for spotting this! I think this should be fixed in the latest commit.